tests/test_svg.py

```diff
@@ -174,7 +174,8 @@ def test_collection(self):
         self.assertSVG(GeometryCollection(), '<g />')
         # Valid
         self.assertSVG(
-            Point(7, 3).union(LineString([(4, 2), (8, 4)])),
+            GeometryCollection(
+                [Point(7, 3), LineString([(4, 2), (8, 4)])]),
```

Comment:
This is indeed related to OverlayNG (I once tested this, but forgot to do a PR for shapely to fix it). 
The order of the parts of the geometry collection changed